pybacktrip/backends/fuseki.py

```diff
@@ -68,13 +66,19 @@ def triples(self, triple: "Triple") -> "Generator":
         if not variables:
             variables.append("*")
         whereSpec = " ".join(
-            f"?{tripleName}"
-            if tripleValue is None
-            else tripleValue
-            if tripleValue.startswith("<")
-            else "<{}{}>".format(self.__namespaces[""], tripleValue[1:])
-            if tripleValue.startswith(":")
-            else f"<{tripleValue}>"
+            (
+                f"?{tripleName}"
+                if tripleValue is None
+                else (
+                    tripleValue
+                    if tripleValue.startswith("<")
+                    else (
+                        "<{}{}>".format(self.__namespaces[""], tripleValue[1:])
```

Comment:
It is possible to use f-strings to slightly simplify this:
```suggestion
                        f"<{self.__namespaces['']}{tripleValue[1:]}>"
```